new-client/src/plugins/collector/collector.js

```diff
@@ -28,8 +28,7 @@ class Collector extends React.Component {
         serviceConfig: this.serviceConfig
       }
     });
-
-    this.collectorModel.app = props.app;
+// Remove line
```

Comment:
```suggestion
```

Reply:
I noticed and fixed it in another commit :)